test/ch/imedias/rsccfx/model/SystemCommanderTest.java

```diff
@@ -0,0 +1,48 @@
+package ch.imedias.rsccfx.model;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests the SystemCommander Class.
+ */
+public class SystemCommanderTest {
+
+  SystemCommander systemCommander;
+
+  /**
+   * Initializes test fixture before each test.
+   */
+  @Before
+  public void setUp() throws Exception {
+    systemCommander = new SystemCommander();
+  }
+
+  /**
+   * Test for {@link SystemCommander#executeTerminalCommand(String)}.
+   */
+  @Test
+  public void testExecuteTerminalCommandIllegalArguments() throws Exception {
+    try {
+      systemCommander.executeTerminalCommand(null);
+      fail("IllegalArgumentException was expected when Command is null");
+    } catch (IllegalArgumentException e) {
+      // expected behavior
+    }
+  }
+
+  /**
+   * Test for {@link SystemCommander#executeTerminalCommand(String)}.
+   * Tests method with standard Linux shell-command "echo"
+   */
+  @Test
+  public void testExecuteTerminalCommand() throws Exception {
+    String teststring = "testExecuteTerminalCommand";
+    StringBuilder command = new StringBuilder("echo ").append(teststring);
+    String output = systemCommander.executeTerminalCommand(command.toString());
+    assertEquals(teststring, output);
+  }
```

Comment:
Renamed it to "testTerminalCommand", hope that's okay - otherwise feel free to revert :)